### PR TITLE
Reworked movie module to rely on TMDB.

### PR DESCRIPTION
**Pull request by edadesd**
Addresses issue #242  by making the module rely on the TMDB API instead of OMDB. TMDB requires attribution by placing a logo and a non-endorsement message. The logo was placed in the movie module's source box in the README, while the non-attribution message is in the module's documentation.

@swapagarwal will need to sign up for a TMDB account for the app at: https://www.themoviedb.org/account/signup - the instructions for getting an API key can be found at https://www.themoviedb.org/faq/api?language=en

### CODE REVIEW

File: modules/src/movie.py [hunk 1]
```diff
@@ -1,22 +1,46 @@
...
         movie = entities['movie'][0]['value']
+
+        # Make a search request to the API to get the movie's TMDB ID.
+        # TMDB required attribution notice: "This product uses the TMDb API but is not
+        # endorsed or certified by TMDb."
+
         with requests_cache.enabled('movie_cache', backend='sqlite', expire_after=86400):
-            r = requests.get('http://www.omdbapi.com/?t=' + movie + '&plot=full&r=json')
+            request_str = 'https://api.themoviedb.org/3/search/movie?api_key='
+            request_str += config.TMDB_API_KEY + '&query=' + movie
```
Comment (swapagarwal) [inline modules/src/movie.py]: It'd be cleaner to use `params` parameter in `requests.get()` method to generate the query string. You can refer to the `lyrics` module: https://github.com/swapagarwal/JARVIS-on-Messenger/blob/master/modules/src/lyrics.py#L17-L20

**Comment by swapagarwal** [comment]
@edadesd I've signed up for a TMDb account, and added the api key on server. 😄

**Comment by swapagarwal** [comment]
Also, 👍 for following the attribution guidelines!

**Comment by edadesd** [comment]
Made the requested changes.

**Comment by swapagarwal** [comment]
Thanks Daniel! 🎉

**Pull request merged**